src/supervisor-api.ts

```diff
@@ -79,6 +79,10 @@ interface SupervisorAPIConstructOpts {
 	healthchecks: Array<() => Promise<boolean>>;
 }
 
+interface SupervisorAPIStopOpts {
```

Comment:
I created this interface so in the future if we want to pass more options for when we are closing the SupervisorAPI we can. Otherwise I could have just passed a boolean to the close function but thought this was more robust.